src/components/StorageOverview/Inventory/tests/DiskInventoryUtils.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render } from 'enzyme';
+
+import { Inventory } from '../Inventory';
+import { default as InventoryFixtures } from '../fixtures/Inventory.fixture';
+
+const testInventoryOverview = () => <Inventory {...InventoryFixtures[0].props} />;
```

Comment:
this is not a test for `DiskInventoryUtils` but for an `Inventory` component.

Reply:
Ack, sorry! Stupid mistake :/